README.md

```diff
@@ -840,6 +841,185 @@ const audits = await descopeClient.management.audit.search({ actions: ['LoginSuc
 console.log(audits);
 ```
 
+### Manage Authz
+
+Descope support full relation based access control (ReBAC) using a zanzibar like schema and operations.
+A schema is comprized of namespaces (entities like documents, folders, orgs, etc.) and each namespace has relation definitions to define relations.
+Each relation definition can be simple (either you have it or not) or complex (union of nodes).
+
+A simple example for a file system like schema would be:
```

Comment:
just making sure I understand this sentence (and the context of the yaml file bellow) - the yaml file doesn't have any usage from the developer perspective - it is just a readable presentation